app/src/main/java/io/gnosis/safe/ui/settings/view/AddressItem.kt

```diff
@@ -11,16 +12,17 @@ import io.gnosis.safe.R
 import io.gnosis.safe.databinding.ViewAddressItemBinding
 import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
 import pm.gnosis.model.Solidity
-import pm.gnosis.svalinn.common.utils.copyToClipboard
-import pm.gnosis.svalinn.common.utils.getColorCompat
-import pm.gnosis.svalinn.common.utils.openUrl
-import pm.gnosis.svalinn.common.utils.snackbar
+import pm.gnosis.svalinn.common.utils.*
+import timber.log.Timber
 
 class AddressItem @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
+    init {
```

Comment:
I would add a new line before init here and reorder properties so that public ones are at the top and private ones follow

Reply:
Wait. Privates first is what we do in other classes. See: EndpointItem etc.